eclipse_development_process.adoc

```diff
@@ -310,182 +310,180 @@ digraph {
+_(Most of this section is paraphrased from the excellent http://www.apache.org/dev/Release.html[Apache Software Foundation Releases FAQ]. The Eclipse community has many of the same beliefs about Releases as does the Apache community and their words were already excellent. The Apache Software Foundation Releases FAQ is distributed under the http://www.apache.org/licenses/LICENSE-2.0[Apache License, Version 2.0].)_
 
-Releases are, by definition, anything that is distributed outside of the committers of a project. If consumers are being directed to download a build, then that build has been released (modulo the exceptions below). All projects and committers must obey the Eclipse Foundation requirements on approving any release.
+Releases are, by definition, anything that is distributed outside of the Committers of a Project. If consumers are being directed to download a build, then that build has been Released (modulo the exceptions below). All Projects and Committers must obey the Eclipse Foundation requirements on approving any Release.
 
-The {ipPolicyUrl}[Eclipse IP Policy] must be followed at all times. Prior to issuing any release, the Project Lead must assert that the IP Policy has been followed and all approvals have been received.
+The {ipPolicyUrl}[Eclipse IP Policy] must be followed at all times. Prior to issuing any Release, the Project Lead must assert that the IP Policy has been followed and all approvals have been received.
 
-A project can make official releases for one calendar year following a successful link:#6_3_5_Progress_Review[progress review] or link:#6_3_3_Release_Review[release review]. The project leadership chain may--at its discretion--require that the project engage in additional reviews (e.g. a progress or release review) prior to issuing a release.
+The Project Team must provide (either directly or indirectly) a link between the distribution form of Release artifacts and the corresponding source code.
 
-*Exception 1: nightly and integration builds* During the process of developing software and preparing a release, various nightly and integration builds are made available to the developer community for testing purposes. Do not include any links on the project website, blogs, wikis, etc. that might encourage non-early-adopters to download and use nightly builds, release candidates, or any other similar package (links aimed at early-adopters and the project's developers are both permitted and encouraged). The only people who are supposed to know about such packages are the people following the developer mailing list and thus are aware of the limitations of such builds.
+A Project can make official Releases for one calendar year following a successful link:#6_3_5_Progress_Review[Progress Review] or link:#6_3_3_Release_Review[Release Review]. The Project Leadership Chain may--at its discretion--require that the Project engage in additional Reviews (e.g. a progress or Release Review) prior to issuing a Release.
```

Comment:
Is this a special formatting with the two dashes `--` and no space separation?

Reply:
That converts into an em-dash. It is grammatically correct to **not** put spaces around an em dash.